__tests__/app.test.js

```diff
@@ -65,23 +65,83 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
+                result.forEach((element) => {
+                    if (result.length != 0) {
```

Comment:
This if is in the wrong place - you should expect result.length to be greater than 0 BEFORE your forEach, because it won't have any effect when it's inside the forEach. The length would have to be not zero for this line to run in the first place